<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutApp</class>
 <widget class="QMainWindow" name="ShortcutApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="searchB">
     <property name="geometry">
      <rect>
       <x>354</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="editB">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>EDIT</string>
     </property>
    </widget>
    <widget class="QComboBox" name="shortcut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shortcut App</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
